systemtest/src/main/resources/StrimziUpgradeST.json

```diff
@@ -257,5 +257,42 @@
       "flakyEnvVariable": "none",
       "reason" : "Test is working on all environment used by QE."
     }
+  },
+  {
```

Comment:
AFAIU this, this block should basically just replace the previous block. Not be there in addition. Could you please confirm @Frawless?

Reply:
Yes. Just update `proceduresAfter` and image names correctly in that case.